pkgs/development/libraries/vulkan-loader/default.nix

```diff
@@ -50,4 +50,6 @@ stdenv.mkDerivation (finalAttrs: {
     broken = finalAttrs.version != vulkan-headers.version;
     pkgConfigModules = [ "vulkan" ];
   };
+} // lib.optionalAttrs stdenv.isi686 {
+  NIX_CFLAGS_COMPILE = "-D_FILE_OFFSET_BITS=64";
```

Comment:
Please inline this. Things behind meta are very often overlooked

Reply:
It's to avoid rebuilds, we can set it globally if we decide it's a good idea.